Makefile

```diff
@@ -339,6 +340,9 @@ platform-linux-riscv64:
 platform-linux-s390x:
 	@$(MAKE) --no-print-directory GOOS=linux GOARCH=s390x NPMDIR=npm/esbuild-linux-s390x platform-unixlike
 
+platform-linux-loong64:
+	@$(MAKE) --no-print-directory GOOS=linux GOARCH=loong64 NPMDIR=npm/esbuild-linux-loong64 platform-unixlike
```

Comment:
> * For now, only `Go@1.18.2` and newer suppoort Loongarch.

When I try to run this command, I get this error:

```
go: unsupported GOOS/GOARCH pair linux/loong64
```

My `go version`:

```
go version go1.18.3 darwin/arm64
```

That implies that Go 1.18.2 and newer do not support this architecture. This makes sense because it's [supposed to only be supported in Go 1.19 and up](https://tip.golang.org/doc/go1.19):

>  Go 1.19 adds support for the Loongson 64-bit architecture LoongArch on Linux (`GOOS=linux`, `GOARCH=loong64`).